addon/components/vertical-collection/component.js

```diff
@@ -29,18 +29,54 @@ const VerticalCollection = Component.extend({
 
   tagName: 'vertical-collection',
 
+  /**
+   * Property name used for storing references to each item in items. Accessing this attribute for each item
+   * should yield a unique result for every item in the list.
+   * 
+   * @property key
+   * @type String
+   * @default '@identity'
+   */
   key: '@identity',
 
   // –––––––––––––– Required Settings
 
+  /**
+   * Estimated height of an item to be rendered. Use best guess as this will be used to determine how many items
+   * are displayed virtually, before and after the vertical-collection viewport.
+   *
+   * @property estimateHeight
+   * @type Number
+   * @required
+   */
   estimateHeight: null,
 
-  // usable via {{#vertical-collection <items-array>}}
+  /**
+   * List of objects to svelte-render.
+   * Can be called like `{{#vertical-collection <items-array>}}`, since it's the first positional parameter of this component.
+   *
+   * @property items
+   * @type Array
+   * @required
+   */
   items: null,
 
   // –––––––––––––– Optional Settings
+  /**
+   * Indicates if this component's height will change after initial render.
+   * If true, item tracking will be done using the StaticRadar, otherwise the DynamicRadar.
+   * @property staticHeight
+   * @type Boolean
+   */
   staticHeight: false,
 
+  /**
+   * Indicates whether or not list items in the Radar should be reused on update of virtual components (e.g. scroll).
+   * This yields performance benefits because it is not necessary to repopulate the component pool of the radar.
+   * // WHEN WOULD WE WANT TO SET THIS TO FALSE?
```

Comment:
Set to `false` when recycling a component instance has undesirable ramifications including:

- use of `unbound` by the component or a sub-component
- use of `init` for instance state by the component or a sub-component that differs between instances (can move to `didInitAttrs` to fix this)
- the templates for individual items vary widely or are based on conditionals that are likely to change (ie would defeat any benefits of DOM recycling anyway)